### PR TITLE
Fix: Hover animation on tabs is not working as expected in Tabs widget (#20795) [ED-9162]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [x] I have added unittests to verify the code works as intended
- [x] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by TzviRabinovitch — DISMISSED** [review]

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 1]
```diff
@@ -727,10 +727,25 @@ test.describe( 'Nested Tabs tests', () => {
...
+		await expect( tabHover ).toBe( true );
+
+		// Test the active tabs with hover.
+		await page.locator( '.e-normal.e-active' ).hover();
+		await expect( page.locator( '.e-normal.e-active' ) ).toHaveCSS( 'transform', 'none' );
+
+		// Test on mobile.
 		await page.setViewportSize( viewportSize.mobile );
 		await expect( page.locator( '.e-collapse.e-active' ) ).toHaveClass( 'e-n-tab-title e-collapse elementor-animation-grow e-active' );
-		await page.setViewportSize( viewportSize.desktop );
```
Comment (davseve) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:732]: What is the reason for this change?
Did you move this line down just for the file order?

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 2]
```diff
@@ -727,10 +727,25 @@ test.describe( 'Nested Tabs tests', () => {
 		await page.waitForSelector( '.elementor-widget-n-tabs' );
 		// Test on desktop.
 		await expect( page.locator( '.e-normal.e-active' ) ).toHaveClass( 'e-n-tab-title e-normal elementor-animation-grow e-active' );
+
+		// Test the hover animation.
+		await page.locator( '.e-normal:not( .e-active )' ).last().hover();
+		const tabHover = await page.locator( '.e-normal:not( .e-active )' ).last().evaluate( ( element ) => {
```
Comment (davseve) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:733]: You have duplicated code in the two lines above, you can use set a const with the last not active tab and use.
Something like:

```
const lastNotActiveTab = page.locator( '.e-normal:not( .e-active )' ).last();
		await lastNotActiveTab.hover();
		const tabHover = await lastNotActiveTab.evaluate( ( element ) => {
			const animationValue = window.getComputedStyle( element ).getPropertyValue( 'transform' );
			
```
You can add it to the test`Arrange` section.

**Review by davseve — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js (hunk 2 above)
Comment (hein-obox) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js]: Updated

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js (hunk 1 above)
Comment (hein-obox) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:732]: Changed.

**Review by TzviRabinovitch — DISMISSED** [review]

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js (hunk 2 above)
Comment (davseve) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:733]: Can't you use `const tabHover = await tabNormal.evaluate( ( element ) => {`?

**Review by davseve — CHANGES_REQUESTED** [review]

**Review by davseve — APPROVED** [review]

**Review by TzviRabinovitch — APPROVED** [review]
